e2e/machineset_migration_test.go

```diff
@@ -188,21 +189,21 @@ var _ = Describe("[sig-cluster-lifecycle][OCPFeatureGate:MachineAPIMigration] Ma
 						cl,
 						[]*capiv1beta1.MachineSet{capiMachineSet},
 						[]*capav1.AWSMachineTemplate{awsMachineTemplate},
-						[]*machinev1beta1.MachineSet{mapiMachineSet},
+						[]*mapiv1beta1.MachineSet{mapiMachineSet},
 					)
 				})
 			})
 
 			It("should find MAPI MachineSet .status.authoritativeAPI to equal CAPI", func() {
-				Eventually(komega.Object(mapiMachineSet)).Should(HaveField("Status.AuthoritativeAPI", Equal(machinev1beta1.MachineAuthorityClusterAPI)))
+				Eventually(komega.Object(mapiMachineSet)).Should(HaveField("Status.AuthoritativeAPI", Equal(mapiv1beta1.MachineAuthorityClusterAPI)))
```

Comment:
Should we put timeouts on these eventually calls? So we stay consistent thoughout the file?